blueocean-personalization/src/main/js/components/DashboardCards.jsx

```diff
@@ -94,7 +94,9 @@ export class DashboardCards extends Component {
           cards={pausedCards}
         />) : null;
         const favoriteCardsStack = (<StackOutput
-          message={t('dashboardCard.input.favorite')}
+          message={t('dashboardCard.input.favorite', {              
```

Comment:
I believe there is another card stack that may need a default value for its label as well.